server/methods/registerUser.js

```diff
@@ -97,3 +97,9 @@ Meteor.methods({
 		return userId;
 	},
 });
+
+RateLimiter.limitMethod('registerUser',
```

Comment:
Since the value came from a setting you will need to watch the setting and on change remove the limiter and recreate it.